release-notes-1.2.0.md

```diff
@@ -0,0 +1,751 @@
+
+**Command:**
+```sh
+ballerina bindgen [(-cp|--classpath) <classpath>...]
+                  [(-o|--output) <output>]
+                  (<class-name>...)
+```
+
+**Options:**
+1. `(-cp|--classpath) <classpath>...`: One or more comma-delimited classpaths for obtaining the JAR files required for generating the Ballerina bindings. This is not needed if the Ballerina bridge code is to be generated for standard Java classes.
+2. `(-o|--output) <output>`: Location for the generated Ballerina bridge code. If this path is not specified, the output will be written to the same directory from where the command is run.
+3, `<class-name>...`: One or more space-separated fully-qualified Java class names for which the bridge code is to be generated.
+
+### Testerina
+
+#### Function Mocking support
+
+The Ballerina test framework provides the capability to mock a function. By using the mocking feature, you can easily mock a function in a module that you are testing or a function of an imported module. This feature will help you to test your Ballerina code independently from other modules and functions. Function mocking is only supported for project-based execution in v1.2.0.
+
+The function specified with the `@test:Mock {}` annotation will be considered as a mock function, which gets triggered every time the original function is called. The original function that will be mocked should be defined using the annotation parameters.
+
+#### Test report
+
+The `ballerina build` and `ballerina test` commands now generate an HTML report at the end of test execution. This would contain the total test cases executed, passes, failures, and skipped tests with project-level, module-level, and individual test details. The link to the test report can be found at the end of the test execution.
+
+#### Code coverage
+
+Introduced the `--code-coverage` flag for `ballerina build` and `ballerina test` commands. When the flag is provided, an additional section with code coverage details will be displayed in the test report. This would contain coverage details at project-level, module-level, and of individual files. 
+
+>**Note:** This is only supported with project-based execution in v1.2.0.
+
+## IDE Plugins & Language Server
+
+### Go to Definition support for Standard Library content
+
+You can go to [The Standard library](https://ballerina.io/v1-2/learn/api-docs/ballerina/) content such as functions, action invocations, types, etc. by executing the `Go to Definition` action.
+
+### New code actions
+
+- All imports are optimized
+- Abstract functions are implemented
```

Comment:
Would be good if these two can be elaborated bit. The second one especially since it doesn't really convey anything at all to the average user. Points to be covered:
- What does it do?
- How do I use it?

Reply:
Please send further comments/suggestions in a PR as this is already merged to adhere to the release deadlines.